vendor/firebase/shim.js

```diff
@@ -1,9 +1,18 @@
 /* globals Firebase */
 
-define('firebase', [], function() {
-  "use strict";
+(function() {
+  function fakeFirebase() {
+    var error = "Firebase is not available. Either the bower package is missing or your app is in Fastboot Mode.";
+    this.to = function() {
+      throw Error(error);
+    };
+    throw Error(error);
+  }
 
-  return {
-    'default': firebase
-  };
-});
+  function vendorModule() {
+    'use strict';
+    return {'default': self['firebase'] || fakeFirebase};
```

Comment:
Not sure what self is, just was copied from here too https://github.com/runspired/ember-tweenlite/blob/master/vendor/shims/tweenlite.js